.azure/app-cloud-e2e.yml

```diff
@@ -80,10 +83,10 @@ jobs:
 
     - bash: |
         python -m pip install playwright
-        python -m playwright install --with-deps
+        python -m playwright install  # --with-deps
```

Comment:
Disable the dependencies as we use playwright image; maybe skip this step completely?

Reply:
yes, let's try to remove this step completely if we already use playwright image

Reply:
trying now... :pray: